sdk/servicebus/Azure.Messaging.ServiceBus/tests/Transactions/TransactionLiveTests.cs

```diff
@@ -355,6 +355,53 @@ await receiver.CompleteMessageAsync(receivedMessage), Throws.InstanceOf<ServiceB
                 Assert.NotNull(receivedMessage);
                 Assert.AreEqual(message2.Body.ToString(), receivedMessage.Body.ToString());
                 await receiver.CompleteMessageAsync(receivedMessage);
+
+                // validate that the Local transaction cannot span multiple entities error is not thrown when outside of a transaction
+                await using var scope2 = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false);
+                ServiceBusReceiver receiver2 = client.CreateReceiver(scope2.QueueName);
+                await receiver2.ReceiveMessageAsync(TimeSpan.FromSeconds(2));
+            }
+        }
+
+        [Test]
+        public async Task CanUseSameSenderBothWithinAndOutsideTransactionSimultaneously()
+        {
+            await using (var scope1 = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false))
+            {
+                await using var client = CreateClient();
+                ServiceBusSender sender1 = client.CreateSender(scope1.QueueName);
+                ServiceBusReceiver receiver = client.CreateReceiver(scope1.QueueName);
+
+                await using var scope2 = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false);
+                ServiceBusSender sender2 = client.CreateSender(scope2.QueueName);
+
+                ServiceBusMessage message = ServiceBusTestUtilities.GetMessage();
+                await sender1.SendMessageAsync(message);
+
+                ServiceBusReceivedMessage receivedMessage = await receiver.ReceiveMessageAsync();
+                Assert.NotNull(receivedMessage);
+
+                var tcs = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
+
+                _ = SendMessageAsync();
```

Comment:
Should we consider capturing the task and awaiting at the very end of the test, just to force any errors to surface?